prow/scripts/provision-vm-and-start-kyma-k3d.sh

```diff
@@ -183,6 +183,12 @@ if [[ -v TELEMETRY_ENABLED ]]; then
   utils::send_to_vm "${ZONE}" "kyma-integration-test-${RANDOM_ID}" "${SCRIPT_DIR}/cluster-integration/kyma-integration-k3d-telemetry-components.yaml" "~/kyma-integration-k3d-telemetry-components.yaml"
 fi
 
+if [[ -v TELEMETRY_TRACING_ENABLED ]]; then
+  log::info "Copying CRD file for telemetry pipeline"
+  #shellcheck disable=SC2088
+  utils::send_to_vm "${ZONE}" "kyma-integration-test-${RANDOM_ID}" "/home/prow/go/src/github.com/kyma-project/kyma/components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_tracepipelines.yaml" "~/kyma/installation/resources/crds/telemetry/telemetry.kyma-project.io_tracepipelines.yaml"
```

Comment:
Can you try to solve this by copying within the VM itself in `kyma-integration-k3d.sh`? There it's just a local operation since all Kyma sources have been copied to the VM already. And you probably need to copy the generated `role.yaml` from the operator source as well.